en/docs/learn/extensions/extending-scope-validation.md

```diff
@@ -1,65 +1,71 @@
 
-        !!! tip
-    You can also use the curl request listed under the **Generate Access Tokens** section for the steps 3 and 4 based on the grant type.
 
+    [![]({{base_path}}/assets/attachments/103334763/103334764.png)]({{base_path}}/assets/attachments/103334763/103334764.png)
 
-    ![]({{base_path}}/assets/attachments/103334763/103334764.png)
-4.  Use the Base64 encoded value obtained above in the Authorization header when invoking the following command. This is used to get the token by calling the token API.
+4.  Use the Base64 encoded value obtained above in the Authorization header when invoking the following command. 
 
-        !!! tip
-    Make sure you include a random scope in the request which will be any value suitable for the name of the scope.
+     This is used to get the token by calling the token API.
+
+    !!! tip
+        Make sure you include a random scope in the request which will be any value suitable for the name of the scope.
 
 
     ``` xml
-        curl -k -d "grant_type=password&username=admin&password=admin&scope=some_random_scope" -H "Authorization: Basic WmRFUFBvZmZwYVFnR25ScG5iZldtcUtSS3IwYTpSaG5ocEVJYUVCMEN3T1FReWpiZTJwaDBzc1Vh" -H "Content-Type: application/x-www-form-urlencoded" https://10.100.0.3:8243/token
+    curl -k -d "grant_type=password&username=admin&password=admin&scope=some_random_scope" -H "Authorization: Basic WmRFUFBvZmZwYVFnR25ScG5iZldtcUtSS3IwYTpSaG5ocEVJYUVCMEN3T1FReWpiZTJwaDBzc1Vh" -H "Content-Type: application/x-www-form-urlencoded" https://10.100.0.3:8243/token
     ```
 
-    Along with the token, you receive a response from the server similar to the one below.
+     Along with the token, you receive a response from the server similar to the one below.
 
     ``` java
-            {"scope":"default","token_type":"bearer","expires_in":3600,"refresh_token":"23fac44e9b7e1ae95a33b85f4f26decd","access_token":"9474fa104ccb196303f41c8a5ee6f48"}
+    {"scope":"default","token_type":"bearer","expires_in":3600,"refresh_token":"23fac44e9b7e1ae95a33b85f4f26decd","access_token":"9474fa104ccb196303f41c8a5ee6f48"}
     ```
 
-    You may not see the scope you requested for in this response as it has not been whitelisted yet.
+    You may not see the scope you requested for in this response as it has not been allowlisted yet.
 
 5.  Shut down the server.
 
-6.  To whitelist the scope, add the following under the `<OAuthConfigurations>` element in the `<APIM_HOME>/repository/conf/api-manager.xml` file and restart the server.
+6.  To allowlist the scope, add the following under the `<OAuthConfigurations>` element in the `<APIM_HOME>/repository/conf/api-manager.xml` file and restart the server.
 
     ``` xml
-            <ScopeWhitelist>
-                <Scope>^device_.*</Scope>
-                <Scope>some_random_scope</Scope>
-            </ScopeWhitelist>
+    <ScopeWhitelist>
+        <Scope>^device_.*</Scope>
+        <Scope>some_random_scope</Scope>
+    </ScopeWhitelist>
     ```
```

Comment:
This configuration seems outdated because now the deployment.toml is used for this purpose.

Reply:
fixed with 4842dc82deb5a92298d2b59e55835c835a1560e4